crates/kubelet/src/bootstrapping/mod.rs

```diff
@@ -66,14 +66,23 @@ async fn bootstrap_auth<K: AsRef<Path>>(
             })?;
         let server = named_cluster.cluster.server;
         trace!(%server, "Identified server information from bootstrap config");
-        let ca_data = named_cluster
-            .cluster
-            .certificate_authority_data
-            .ok_or_else(|| {
-                anyhow::anyhow!(
-                    "Unable to find certificate authority information in bootstrap config"
-                )
-            })?;
+
+        let ca_data = match named_cluster.cluster.certificate_authority {
+            Some(certificate_authority) => {
+                read_to_string(certificate_authority).await.map_err(|e| {
```

Comment:
do we need to encode the string from the file?

Reply:
If we want to do it this way, then yes, this would need to be base64 encoded